app/iscsi_tgt/iscsi_tgt.c

```diff
@@ -101,7 +103,7 @@ main(int argc, char **argv)
 	opts.config_file = SPDK_ISCSI_DEFAULT_CONFIG;
 	opts.name = "iscsi";
 
-	while ((ch = getopt(argc, argv, "c:de:i:l:m:n:p:qs:t:H")) != -1) {
+	while ((ch = getopt(argc, argv, "c:de:i:l:m:n:p:qs:t:bH")) != -1) {
```

Comment:
Sorry, I didn't notice this on the previous review iteration, but we like to keep the getopt() flags in sorted order (e.g. add b at the beginning of the string) to make it easy to tell if there are duplicates.